bedrock/products/models.py

```diff
@@ -109,3 +113,221 @@ def get_context(self, request, *args, **kwargs):
         context["ftl_files"] = ["products/vpn/resource-center", "products/vpn/shared"]
         context["vpn_available"] = vpn_available_in_country
         return context
+
+
+@register_snippet
+class MonitorCallToActionSnippet(TranslatableMixin):
+    split_heading = models.CharField(
+        max_length=60,
+        blank=False,
+    )
+    split_subheading = models.CharField(
+        max_length=60,
+        blank=True,
+    )
+    split_content = RichTextField(null=True, blank=True, features=["bold", "italic", "link", "ul", "ol"])
+    split_button_text = models.CharField(
+        verbose_name="Link Text",
+        max_length=40,
+        blank=False,
+    )
+    split_button_link = models.URLField(
+        verbose_name="Link URL",
+        blank=False,
+    )
+    split_image = models.ForeignKey(
+        "cms.BedrockImage",
+        null=True,
+        blank=False,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    panels = [
+        MultiFieldPanel(
+            [
+                FieldPanel(
+                    "split_heading",
+                    heading="Intro Heading",
+                ),
+                FieldPanel(
+                    "split_subheading",
+                    heading="Intro Subheading (optional)",
+                ),
+                FieldPanel(
+                    "split_content",
+                    heading="Intro Content (optional)",
+                ),
+                FieldRowPanel(
+                    [
+                        FieldPanel("split_button_text", heading="Intro Button text"),
+                        FieldPanel("split_button_link", heading="Intro Button link"),
+                    ]
+                ),
+                FieldPanel("split_image", heading="Intro Image"),
+            ],
+            heading="Intro Block",
+        ),
+    ]
+
+    class Meta(TranslatableMixin.Meta):
+        verbose_name = "Monitor Call To Action Snippet"
+        verbose_name_plural = "Monitor Call To Action Snippets"
+
+    def __str__(self):
+        return f"{self.split_heading} - CTA Snippet"
+
+
+class MonitorArticleIndexPage(AbstractBedrockCMSPage):
+    split_heading = models.CharField(
+        max_length=60,
+        blank=False,
+    )
+    split_subheading = models.CharField(
+        max_length=60,
+        blank=True,
+    )
+    split_content = RichTextField(null=True, blank=True, features=["bold", "italic", "link", "ul", "ol"])
```

Comment:
Nit: we should move this features list in to a setting  eg `RICHTEXT_FEATURES_MINIMAL`